bin/server.dart

```diff
@@ -10,7 +10,6 @@ void main() async {
     InternetAddress.anyIPv4,
     3030,
     wsMiddleware: [
-      encryptionMiddleware,
```

Comment:
Yes I fully support that idea. I disabled it, because it did not work yet out of the box and I'd like to have it easy to go for the user. I think we should add support for encrypted network traffic in a future release when we can add a good UX (Support in config files / CLI for example) to that.

Reply:
Jup, that sounds fair - we really need a little tracker though so I'll open an issue :)